src/Marian/index.ts

```diff
@@ -291,13 +295,17 @@ export default class Marian {
     if (typeof searchProperty === 'string') {
       searchProperty = [searchProperty];
     }
-
     try {
       const res = await this.index.fetchFacets(query, searchProperty, filters);
       res.facets = sortFacets(res.facets);
       return res;
     } catch (e) {
-      console.error(`Error fetching facet metadata: ${JSON.stringify(e)}`);
+      console.error(
```

Comment:
(Nit) We can probably use `log.error` instead of `console.error` to help consolidate the log formatting